k8s/k8s.go

```diff
@@ -364,7 +364,12 @@ func CmdDelK8s(c *calicoclient.Client, ep api.WorkloadEndpointMetadata, args *sk
 		return ipamErr
 	}
 
-	logger.WithField("endpoint", wep).Info("Teardown processing complete.")
```

Comment:
A nicer fix might be to add the `workloadID` (and probably container ID as well?) to the context logger here: https://github.com/projectcalico/cni-plugin/blob/master/calico.go#L82

And then just turn this into a single log "Teardown processing complete" - we log the wep above if it exists.

Reply:
wdyt?

Reply:
hmm yeah, but I'm wondering if there's a bunch of pod deletion/addition logs will interleave and that will make it hard to know which WEP was torn down. Would that not be the case?

Reply:
Actually, scratch that. `workload` is already part of the context logger for `cmdDel`, we should've just used that. I can also add it (and containerID) to the `cmdAdd` context logger as well